docs/wiki/entities-and-hazards.md

```
@@ -0,0 +1,316 @@
+# Entities & Hazards
+
+#home
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)